db/changelog/v-1/07-changeset-agent-table.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="add-agent-table" author="petertrr" context="dev or prod">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="agent"/>
+            </not>
+        </preConditions>
+        <createTable tableName="agent">
+            <column name="agent_id" type="varchar(64)">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="execution_id" type="bigint">
+                <constraints foreignKeyName="fk_agent_execution" references="execution(id)" nullable="false"/>
+            </column>
+        </createTable>
+
+        <addUniqueConstraint
+                tableName="agent"
+                columnNames="agent_id"/>
```

Comment:
It's not needed. A primaryKey constraint includes a unique constraint

Reply:
ok, removed